cmd/generate/codeowners/codeowners.go

```diff
@@ -176,21 +181,24 @@ func run(opts *Options, cmd *cobra.Command) error {
 		return fmt.Errorf("error traversing git log: %w", err)
 	}
 
-	// Bootstrap codeowners
-	var outputPath string
+	// Add optional output path to base path
+	fullOutputPath := filepath.Join(opts.path, opts.outputPath)
+
+	// Define the file type based on a flag
+	var finalOutputPath string
```

Comment:
Why the var name change from `outputPath` to `finalOutputPath`?

Reply:
Initially, this was to differentiate between the `fullOutputPath` variable.

As I'll go through a rewrite of parsing the `outputPath` properly and not assuming it is under the root of the current repository, this will be removed.